### DISCUSSION TITLE
Can I use civitai model for better output?

### DISCUSSION THREAD

**Discussion by oldhand7** [lllyasviel/Fooocus#3421] [Q&A] [answered]
https://civitai.com/models

Seems there are several new models coming and downloaded
Would like to know if we can use one of them and if so what is the best choice for fooocus in terms of performance & quality.

**Comment by mashb1t** [comment, marked as answer]
best choice are currently the models in https://github.com/lllyasviel/Fooocus?tab=readme-ov-file#default-models, but feel free to create your own presets and use the civitai model URL instead of the HF lllyasviel / mashb1t download URLs.

**Reply by oldhand7** [reply]
Thank you. 
The models in https://github.com/lllyasviel/Fooocus?tab=readme-ov-file#default-models are default models.
I saw lots of new models coming in civitai
Just wondering if we can use better model in fooocus in order for better performance and quality

Appreciate your response.

**Reply by mashb1t** [reply]
> Just wondering if we can use better model in fooocus in order for better performance and quality

Which models are you referring to?

**Reply by oldhand7** [reply]
https://civitai.com/models/119229/zavychromaxl
https://civitai.com/models/352581/vixons-pony-styles



Those models are up to date.
I don't know who made, where it is from, how is it made, how I can adapt those models in Fooocus
As you can see, those models are quite realistic, super quality


And there came out several models coming , seems to be useful in Fooocus.

Thank you for your response.

**Reply by mashb1t** [reply]
Still no indication what sould be "better" about zavychroma compared to any of the default models as it has a distinct look, plus the LoRAs are specifically made for one character, so definitely not broadly applicable as default in Fooocus.
Feel free to create preset(s) yourself using these models/LoRAs.

**Reply by oldhand7** [reply]
When it came to indication, it should be eye, hands fix as well as performance, speed.
Could you let me know your suggestion?

**Reply by mashb1t** [reply]
> best choice are currently the models in https://github.com/lllyasviel/Fooocus?tab=readme-ov-file#default-models

**Comment by kalle07** [comment]
every day 5 to 10 models coming out ^^

eyes and hand ... most is depending on well used hiresfix / upscale  ;)
and performance/speed u can use LCM, Turbo, lightning ... but all ALL these types are not fully capable of the magic deep touch ... and they have less fine-tune possibilities in steps and CFG
